tutorials/phase_contrast_calculation.ipynb

```diff
@@ -0,0 +1,232 @@
+      "22 608.6049584877176 0.3190104363311846 15 31.79650199363629 9.938951771706343 61.17216385900974 61.17216385900974 14.26423649263978  \n",
+      "23 573.206095316669 0.3190104363311846 15 29.94709211339553 10.769232615828514 60.195362865924835 60.195362865924835 13.850859743049933  \n",
+      "24 568.4696277092753 0.3190104363311846 15 29.699635861814023 14.383396290242672 48.22955070063472 48.22955070063472 10.060225130025211  \n",
+      "25 511.6943383627521 0.3190104363311846 15 26.733416846146188 12.771674651652575 45.68986811861396 45.68986811861396 9.921045826507678  \n",
+      "26 447.93899148954324 0.3190104363311846 15 23.40252545972665 9.743591573089361 47.00854945927858 47.00854945927858 11.022679729713788  \n",
+      "27 425.06683870120736 0.3190104363311846 15 22.20757224485278 7.985349785536528 45.59218801930547 45.59218801930547 10.403950129307482  \n",
+      "28 444.72941359335954 0.3402777987532635 16 21.782663812860847 6.764348544180393 43.05250543728471 43.05250543728471 9.76825412804492  \n",
+      "29 411.4806553010514 0.3190104363311846 15 21.49776352321108 6.422468196600676 40.659343004226685 40.659343004226685 9.708810051150099  \n",
+      "30 376.5180457254208 0.3190104363311846 15 19.671145666142305 2.319904025644064 36.06837833672762 36.06837833672762 9.611798980395974  \n",
+      "31 346.10493793057634 0.3190104363311846 15 18.08221605072419 0.7081823870539665 31.08669327199459 31.08669327199459 8.193597605855963  \n",
+      "32 398.67349830567497 0.3402777987532635 16 19.526864019455388 6.959708742797375 25.958488058298826 25.958488058298826 5.767669678381297  \n",
+      "33 404.9680102136374 0.29774307390910565 14 22.66876185658787 6.520148295909166 27.08180920034647 27.08180920034647 5.349701003807637  \n",
+      "34 459.28158497040744 0.3190104363311846 15 23.995118062198163 3.8339455649256706 33.33333555608988 33.33333555608988 8.443373463566209  \n",
+      "35 498.82485517067795 0.29774307390910565 14 27.922555769394553 19.413921404629946 35.482297740876675 35.482297740876675 4.920816476942927  \n",
+      "36 478.38325812824144 0.29774307390910565 14 26.778303177495086 18.437120411545038 33.33333555608988 33.33333555608988 4.886557187900364  \n",
+      "37 428.8996317649215 0.2552083490649477 12 28.009770143777132 18.974360957741737 34.60317684710026 34.60317684710026 4.823598848364521  \n",
+      "38 432.4519824704669 0.2552083490649477 12 28.241760379634798 21.758243788033724 34.55433679744601 34.55433679744601 4.227939026335402  \n",
+      "39 416.9338188620319 0.2552083490649477 12 27.228329349309206 23.418805476278067 32.35653456300497 32.35653456300497 3.138899494197213  \n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print the stats_table dictionary\n",
+    "for each_row in zip(*([i] + (j) for i, j in mapper.stats_table.items())):\n",
+    "    print(*each_row, \" \")\n",
```

Comment:
Could this be made so the column headers line up with the numbers? 

I'd suggest you make a `print_stats` method that makes use of the [tabulate](https://github.com/astanin/python-tabulate) package and does  
`print(tabulate(stats_table, headers='keys', tablefmt='github', floatfmt='.2f'))`

Reply:
I thought in using this tabulate, but I didn't want to add another package/dependency...

Reply:
I just think that at the moment, those numbers are very unclear/unreadable. For the sake of adding a few bytes of dependency they could be rendered useful.